tests/cloudformation/checks/resource/aws/test_LambdaEnvironmentCredentials.py

```diff
@@ -0,0 +1,26 @@
+import os
+import unittest
+
+from checkov.cloudformation.checks.resource.aws.LambdaEnvironmentCredentials import check
+from checkov.cloudformation.runner import Runner
+from checkov.runner_filter import RunnerFilter
+
+
+class TestLambdaEnvironmentCredentials(unittest.TestCase):
+
+    def test_summary(self):
```

Comment:
@mikeurbanski1 can you add a test that will verify the previous FP won't come back?